packages/contracts-bedrock/scripts/upgrades/holocene/scripts/sc-ops-sys-cfg.sh

```diff
@@ -26,6 +33,7 @@ msup render -i "$TASK_DIR/input.json" -o "$TASK_DIR/OVERVIEW.md"
 
 # Generate the README
 sed -i "s/\$SYSTEM_CONFIG_IMPL/$SYSTEM_CONFIG_IMPL/g" "$TASK_DIR/README.md"
+sed -i "s/\$OP_CONTRACTS_RELEASE/$OP_CONTRACTS_RELEASE/g" "$TASK_DIR/README.md"
 
 # Generate the validation doc
```

Comment:
Ideally the validation doc would cover the changes to `baseFeeScalar` and `blobbaseFeeScalar`.

Reply:
Is this something you plan on adding? It would introduce a race condition with another task that that modifies the scalar

Reply:
We should only modify the values on any chain after the SystemConfig has been updated.

Reply:
It's probably easier for us to first use the draft state in this PR, do the `SystemConfig` upgrade on OP Sepolia, look at the resulting state diff, then update the generated verification text in this PR in tandem with upgrading the superchain-ops task.